R/R-BayesDA/Portfile

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           R 1.0
+
+R.setup             cran cran BayesDA 2012.04-1
+revision            0
+categories-append   math
+maintainers         nomaintainer
+license             GPL-2
+description         Functions and datasets for Bayesian Data Analysis (2nd ed.)
+long_description    {*}${description}
+checksums           rmd160  f30a0488c2f2cce64bf4b336f18a770c2cc40d58 \
+                    sha256  4cea1b6a0fe3a6971012902178a739e48004b1a9d4055649eeba210e2b3be23a \
+                    size    37703
+supported_archs     noarch
```

Comment:
I think they are.

I can add `platforms {darwin any}`, but in a separate PR, once for all `noarch` packages. Ideally, it would be better to add in `R` PG, perhaps. Let me think about this a bit.

Reply:
@reneeotten Any idea why it takes so long this time? Nothing of these ports are too heavy on compilation. Most of them do not even build anything.